src/ceph-volume/ceph_volume/main.py

```diff
@@ -9,15 +9,25 @@
 # `entry_points` from `importlib.metadata` does not.
 try:
     from importlib.metadata import entry_points
+    from importlib.metadata import EntryPoints
```

Comment:
I don't think this is importable on python 3.9:
```
$ python3.9
Python 3.9.20 (main, Sep  9 2024, 00:00:00) 
[GCC 14.2.1 20240801 (Red Hat 14.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib.metadata
>>> importlib.metadata.EntryPoints
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'importlib.metadata' has no attribute 'EntryPoints'
>>> 

$ python3.12
Python 3.12.7 (main, Oct  1 2024, 00:00:00) [GCC 14.2.1 20240912 (Red Hat 14.2.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib.metadata 
>>> importlib.metadata.EntryPoint
importlib.metadata.EntryPoint(   importlib.metadata.EntryPoints(
>>> importlib.metadata.EntryPoints
<class 'importlib.metadata.EntryPoints'>
>>> 
```

@guits do we care about supporting python3.9 in ceph-volume?

Reply:
The centos 9 based container images still use python 3.9 and given I think it's unlikely the T release will use a centos 10 based container I feel like we have to support 3.9.

Reply:
> The centos 9 based container images still use python 3.9 and given I think it's unlikely the T release will use a centos 10 based container I feel like we have to support 3.9.

correct

Reply:
Thanks for the heads up. We don't even strictly need to import EntryPoints -- I've removed that import now